src/mudlet-lua/lua/Other.lua

```diff
@@ -700,3 +700,111 @@ function condenseMapLoad()
 
   return loadtime
 end
+
+do
+  -- management things
+
+  -- Dictionary with events as keys and lists of lua functions as values to dispatch events
+  -- to the right functions.
+  local handlers = {}
+
+  -- Remember highest hander ID to avoid ID reuse.
+  local highestHandlerId = 0
+  -- Helps us finding the right event handler from an ID.
+  local handlerIdsToHandlers = {}
+
+  -- C functions that get overwritten.
+  local origRegisterAnonymousEventHandler = registerAnonymousEventHandler
+
+  function registerAnonymousEventHandler(event, func, isOneShot)
+    if type(event) ~= "string"then
+      error(
+        string.format(
+          "Unexpected argument type in function registerAnonymousEventHandler for argument #1. String expected, got %s.",
+          type(event)
+        )
+      )
+    end
+
+    if type(func) ~= "function" and type(func) ~= "string" then
+      error(
+        string.format(
+          "Unexpected argument type in function registerAnonymousEventHandler for argument #2. Function or string expected, got %s.",
```

Comment:
As above:

    "registerAnonymousEventHandler: bad argument #2 type (literal function or function name as string expected, got %s!)"

TBH I am not sure how the argument concerned would look here so I am not sure what to put in the `(`...`)` to follow our "style" for _this_ case.